identity/app/views/profile/smsConsent.scala.html

```diff
@@ -18,7 +19,9 @@
                 @fragments.form.switch(
                     title = "",
                     subheading = None,
-                    description = Some("I would like to receive updates about the Guardian products and services I’ve selected above <b>by SMS</b> (text messages)."),
+                    description = Some(
+                        Consent.CommunicationSms.latestWording.description.getOrElse("")
+                            .replaceFirst("The Guardian products and services I['’]ve selected above", "<b>$0</b>")),
```

Comment:
Realised I used the wrong apostrophe (i.e. single quote) in the library wording - so made the regex here support both.  I didn't want to re-release the library right now for something so small!